cmd/repo_show.go

```diff
@@ -26,18 +32,42 @@ func aptlyRepoShow(cmd *commander.Command, args []string) error {
 		return fmt.Errorf("unable to show: %s", err)
 	}
 
-	fmt.Printf("Name: %s\n", repo.Name)
-	fmt.Printf("Comment: %s\n", repo.Comment)
-	fmt.Printf("Default Distribution: %s\n", repo.DefaultDistribution)
-	fmt.Printf("Default Component: %s\n", repo.DefaultComponent)
-	if repo.Uploaders != nil {
-		fmt.Printf("Uploaders: %s\n", repo.Uploaders)
-	}
-	fmt.Printf("Number of packages: %d\n", repo.NumPackages())
+	if jsonFlag {
```

Comment:
Why is this command different from the others, in that it doesn't have two separate functions to print txt and json?

Reply:
I got caught up on the other commands and forgot to refactor it. I've fix that now.